backend/buildeR.BLL/Services/SynchronizationService.cs

```diff
@@ -85,10 +86,10 @@ public async Task<bool> CheckIfUserHasToken(int userId)
                 return false;
             }
         }
-        public async Task RegisterWebhook(int projectId, string callback)
+        public async Task RegisterWebhook(int projectId)
         {
-            callback = "https://www.bsa-builder.xyz/api/webhooks";
-            callback += $"/{projectId}/github";
+            var apiURL = Environment.GetEnvironmentVariable("BaseUrl");
```

Comment:
Could you replace it to IConfiguration?

Reply:
Should I set up BaseURL in appsettings? It was not setted yet